### ISSUE TITLE
Use inline command arguments

### ISSUE THREAD

**Issue by MasterOfTheTiger** [amazinigmech2418/SB3-To-JSON#1]
Instead of asking for the filename after you start the script, you can make it much more convenient (and follow standard commands) by using inline cli arguments. Currently it works like this:
```
$ ./sb3_decompress.sh
File Name: my-project.sb3
$ 
```
It would be much easier (and would work better with shells) to do it like this:
```
$ ./sb3_decompress.sh my-project.sb3
```
I believe the code would look like this:
```
$file = $1
cat $file > sb3_decompress_gzip.gz
zcat sb3_decompress_gzip.gz > $file
```

**Comment by MasterOfTheTiger** [comment]
@amazinigmech2418 What do you think of this?

**Comment by AmazingMech2418** [comment]
This is a great idea!

**Comment by MasterOfTheTiger** [comment]
Resolved in 7f7de18ad898e143d3b23fe4ca8620dd27362ecb
